pkg/deploying/deployer.go

```diff
@@ -144,6 +144,8 @@ func (d *Deployer) updateService(desiredObj, runtimeObj *unstructured.Unstructur
 	}
 
 	if !apiequality.Semantic.DeepDerivative(desiredService.Spec, runtimeService.Spec) {
+		desiredService.ObjectMeta.ResourceVersion = runtimeService.ObjectMeta.ResourceVersion
```

Comment:
I wonder if we need to update these field values. `ResourceVersion` and `ClusterIP ` should update via apiserver?